config/env_blank.json

```diff
@@ -38,5 +38,11 @@
     "user": "",
     "password": ""
   },
-  "APPEND_TABLE_NAMES": ["job_run", "data_source_status"]
+  "APPEND_TABLE_NAMES": [
+    "job_run", "data_source_status",
+    "mivideo_media_started_hourly"
+  ],
+  "mivideo": {
```

Comment:
Can we at least make the highest level variable `ALL_CAPS`?

Reply:
Also, I was trying to keep the data fetching variables higher up and the output-related stuff at the bottom. If you're amenable, maybe this can go under the `ZOOM_CONFIG`?

Reply:
Thinking…

Reply:
OK.  I've changed the case and the position.  

Some tools that clean up JSON may change the position of the various properties.  If we really want to keep the data source properties together and the output properties together, it'd be better to make `dataSource` and `output` objects and nest the current properties within them.

Reply:
That makes sense. Let's defer resolving that until we figure out what's happening with issue #29.